mungegithub/features/repo-updates.go

```diff
@@ -157,6 +176,27 @@ func (o *RepoInfo) walkFunc(path string, info os.FileInfo, err error) error {
 	return nil
 }
 
+// normalizeUsers normalizes normalizes all entries in
+// the assignment configuration so that we can do case-
+// insensitive operations on the entries
+func (c *assignmentConfig) normalizeUsers() {
+	c.Approvers = normalizeUsers(c.Approvers)
+	c.Assignees = normalizeUsers(c.Assignees)
+	c.Reviewers = normalizeUsers(c.Reviewers)
+}
+
+func normalizeUsers(users []string) []string {
+	normalizedUsers := users[:0]
```

Comment:
Inspired by the [Go wiki](https://github.com/golang/go/wiki/SliceTricks#filtering-without-allocating)